prody/proteins/blastpdb.py

```diff
@@ -269,8 +283,9 @@ def showSequenceTree(hits):
     """
     clustalw = which('clustalw')
     if clustalw is None:
-        print("The executable for clustalw does not exists, install or add clustalw to path.")
-        return
+        try: clustalw = which('clustalw2')
+        except: raise ValueError("The executable for clustalw does not exist, \
```

Comment:
I don't know who wrote the original code but this is surely not ProDy's style. Nice catch! 
One suggestion is that `ValueError` might not be appropriate here. I would suggest either `EnvironmentError` or `IOerror`. In any case, `RuntimeError` is always an option. Please refer to the [documentation](https://docs.python.org/2/library/exceptions.html).

Additionally, I don't understand why "hits.fasta" can be read before calling the clustalw. Do you know?

Reply:
It writes the file "hits.fasta" from the hits so that clustalw can read it in. I changed inp to f_out to make that clearer.